components/org.wso2.carbon.identity.api.server.organization.configs/org.wso2.carbon.identity.api.server.organization.configs.v1/src/main/resources/organization-configs.yaml

```diff
@@ -0,0 +1,191 @@
+              schema:
+                $ref: '#/components/schemas/Config'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+    post:
+      tags:
+        - Discovery
+      summary: Add organization discovery configuration.
+      description: |
+        This API provides the capability to add discovery configuration for the primary organization. <br>
+          <b>Permission required:</b> <br>
+            * /permission/admin/manage/identity/configmgt/add <br>
+          <b>Scope required:</b> <br>
+            * internal_config_mgt_add
+      operationId: createDiscoveryConfig
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Config'
+      responses:
+        '201':
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Config'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '409':
+          $ref: '#/components/responses/Conflict'
+        '500':
+          $ref: '#/components/responses/ServerError'
+    delete:
+      tags:
+        - Discovery
+      summary: Delete organization discovery configuration.
+      description: |
+        This API provides the capability to delete discovery configuration of the primary organization.<br>
+          <b>Permission required:</b> <br>
+            * /permission/admin/manage/identity/configmgt/delete <br>
+          <b>Scope required:</b> <br>
+            * internal_config_mgt_delete
+      operationId: deleteDiscoveryConfig
+      responses:
+        '204':
+          $ref: '#/components/responses/NoContent'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+components:
+  schemas:
+    Error:
+      type: object
+      required:
+        - code
+        - message
+      properties:
+        code:
+          type: string
+          example: OCM-00000
+          description: An error code.
+        message:
+          type: string
+          example: Some Error Message
+          description: An error message.
+        description:
+          type: string
+          example: Some Error Description
+          description: An error description.
+        traceId:
+          type: string
+          example: e0fbcfeb-3617-43c4-8dd0-7b7d38e13047
+          description: An error trace identifier.
+
+    Config:
+      type: object
+      properties:
```

Comment:
shouldn't this "properties" attribute marked as a required property of "Config" schema?

Reply:
fixed